warehouse/templates/email/admin-organization-renamed/subject.txt

```diff
@@ -0,0 +1,17 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/subject.txt" %}
+
+{% block subject %}{% trans organization_name=organization_name previous_organization_name=previous_organization_name %}Organization renamed from "{{ previous_organization_name }}" to "{{ organization_name }}"{% endtrans %}{% endblock %}
```

Comment:
```suggestion
{% block subject %}{% trans organization_name=organization_name, previous_organization_name=previous_organization_name %}Organization renamed from "{{ previous_organization_name }}" to "{{ organization_name }}"{% endtrans %}{% endblock %}
```

Reply:
Was causing failure when testing renaming locally.
<img width="1120" alt="setup-for-org-name-change-failure" src="https://user-images.githubusercontent.com/1200832/169857463-dd31db29-341e-4c02-8e30-4d87f48d8e60.png">
<img width="1120" alt="error-changing-org-name" src="https://user-images.githubusercontent.com/1200832/169857479-a614485b-061f-4909-9782-4804af67b707.png">

Reply:
This still needs to be updated as well. There were two places with the missing comma.